src/pybind/mgr/devicehealth/module.py

```diff
@@ -450,6 +462,7 @@ def _get_device_metrics(self, devid, sample=None, min_sample=None):
                 except rados.Error as e:
                     self.log.exception("RADOS error reading omap: {0}".format(e))
                     raise
+        ioctx.close()
```

Comment:
does the with statement above not close the ioctx?

Reply:
missed that!